app/controllers/general.php

```diff
@@ -289,19 +291,64 @@
         ->noContent();
 }, ['request', 'response']);
 
-App::error(function ($error, $utopia, $request, $response, $layout, $project) {
+App::error(function ($error, $utopia, $request, $response, $layout, $project, $logger, $user, $loggerBreadcrumbs) {
     /** @var Exception $error */
     /** @var Utopia\App $utopia */
     /** @var Utopia\Swoole\Request $request */
     /** @var Appwrite\Utopia\Response $response */
     /** @var Utopia\View $layout */
     /** @var Appwrite\Database\Document $project */
+    /** @var Utopia\Logger\Logger $logger */
+    /** @var Appwrite\Database\Document $user */
+
```

Comment:
Let's add the phpdoc for `$loggerBreadcrumbs` as well

Reply:
Added